pkg/wshrpc/wshremote/wshremote.go

```
@@ -186,37 +181,337 @@
+				}
+			}
+		}
+	} else if !errors.Is(err, fs.ErrNotExist) {
+		return fmt.Errorf("cannot stat destination %q: %w", destPath, err)
+	}
+	log.Printf("copying %q to %q\n", srcUri, destPath)
+	timeout := time.Millisecond * 100
+	if opts.Timeout > 0 {
+		timeout = opts.Timeout
+	}
+	readCtx, _ := context.WithTimeout(ctx, timeout)
```

golangci-lint (1.62.2)

lostcancel: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak

(govet)